epoxy-preloadersample/src/main/java/com/airbnb/epoxy/preloadersample/ImageModel.kt

```diff
@@ -0,0 +1,39 @@
+package com.airbnb.epoxy.preloadersample
+
+import android.widget.ImageView
+import android.widget.TextView
+import com.airbnb.epoxy.EpoxyAttribute
+import com.airbnb.epoxy.EpoxyModelClass
+import com.airbnb.epoxy.EpoxyModelWithHolder
+import com.bumptech.glide.RequestManager
+
+@EpoxyModelClass(layout = R.layout.list_item)
+abstract class ImageModel : EpoxyModelWithHolder<ImageHolder>() {
+
+    @EpoxyAttribute(EpoxyAttribute.Option.DoNotHash)
```

Comment:
instead of passing glide as an attribute it could be injected in the view holder
```
class ImageHolder {
    val glide by lazy { Glide.with(image.context)

Reply:
Shouldn't we use activity or fragment context with a recyclerview of any type?
Maybe I'm wrong...  

https://android.jlelse.eu/using-glide-few-tips-to-be-a-pro-60f41e29d30a
https://stackoverflow.com/questions/31964737/glide-image-loading-with-application-context/32887693#32887693

Reply:
The context off of the view is the activity context

Reply:
ah... ok